CHANGES

```diff
@@ -34,6 +34,8 @@ Features added
 * #7840: i18n: Optimize the dependencies check on bootstrap
 * #5208: linkcheck: Support checks for local links
 * #5090: setuptools: Link verbosity to distutils' -v and -q option
+* #6698: doctest: Add ``:trim-flags:`` and ``:no-trim-flags:`` options to
+  doctest, testcode and testoutput directives
```

Comment:
I found it hard to understand "flag to trim flags".
Also, I think the only time this flag is used is rarely such as explaining doctest itself.

So, I suggest names as: `:trim-doctest-flags:` and `:no-trim-doctest-flags:`

Reply:
Reasonable.